apps/sim-engine/lib/execution/src/package/simulation/state/behavior_execution/package.js

```diff
@@ -102,7 +107,7 @@ const load_behaviors = (experiment, behavior_descs) => {
         trace.msg;
       throw new Error(JSON.stringify(trace));
     }
-
+     
```

Comment:
Sorry, editor doesn't seem to be set up with autoformatting right now for some reason

```suggestion

```